### ISSUE TITLE
[BUG] Warning appears when cloning a repository

### ISSUE THREAD

**Issue by wate** [Leantime/leantime#2278]
*If you have problems with the installation please use our [community forum](https://community.leantime.io)*

**What set up are you using**
* [x] Self Hosted

**Describe the bug**

The following warning appears when cloning a repository

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'public/assets/images/Screenshots/Blueprints.png'
  'public/assets/images/Screenshots/blueprints.png'
  'public/assets/images/Screenshots/Calendar.png'
  'public/assets/images/Screenshots/calendar.png'
  'public/assets/images/Screenshots/Docs.png'
  'public/assets/images/Screenshots/docs.png'
  'public/assets/images/Screenshots/Goals.png'
  'public/assets/images/Screenshots/goals.png'
  'public/assets/images/Screenshots/ProjectDashboard.png'
  'public/assets/images/Screenshots/projectDashboard.png'
  'public/assets/lottie/leo/Pirate.json'
  'public/assets/lottie/leo/pirate.json'
```

**To Reproduce**

Clone the repository on an OS with a case-insensitive file system
(macOS, Windows etc)

**Expected behavior**

Can be cloned with no warning.

**Leantime Version**

v3.0.4

**Comment by marcelfolaron** [comment]
Fixed in 3.1.0-beta

**Comment by wate** [comment]
@marcelfolaron 

Even v3.1.0-beta shows the following, so the fix seems to be incomplete

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'public/assets/lottie/leo/Pirate.json'
  'public/assets/lottie/leo/pirate.json'
```
